Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -256,15 +256,18 @@
   - type: PressureProtection
     highPressureMultiplier: 0.3
     lowPressureMultiplier: 1000
+  - type: ExplosionResistance
+    damageCoefficient: 0.4
   - type: Armor
     modifiers:
       coefficients:
-        Blunt: 0.8
-        Slash: 0.8
-        Piercing: 0.7
+        Blunt: 0.6
+        Slash: 0.6
+        Piercing: 0.6
+        Caustic: 0.5
```

Comment:
Why more protection than a security suit? The whole point of a brig medic suit is that it shouldn't slow you down, but it shouldn't have the same protection as a security suit.

Reply:
Farewell, i could make it light weight with less armor. Besides, it was weaker and considerably slower than the hard suit prior

Reply:
i agree, this actually makes a lot more sense, just make brigmedic be like standard sec suit and make it maybe 5% faster